blog/en/blog/2023/07/07/tiny-apisix-plugin.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "A \"Tiny\" APISIX Plugin"
+authors:
+  - name: Navendu Pottekkat
+    title: Author
+    url: https://github.com/navendu-pottekkat
+    image_url: https://avatars.githubusercontent.com/u/49474499
+keywords:
+  - Wasm
+  - Go
+  - Plugin
+description: "A \"tiny\" example to demonstrate how Apache APISIX supports Wasm plugins."
+tags: [Plugins]
+image: https://static.apiseven.com/uploads/2023/06/27/Z8CkI8kj_wasm-cover.png
+---
+
+> In this article, we will write a "tiny" Go plugin for APISIX, compile it to a Wasm binary, run it in APISIX, and learn how it all works. We will also compare the benefits and costs of using Wasm plugins, external plugins (plugin runners), and native Lua plugins.
+
+<!--truncate-->
+
+A key feature of Apache APISIX is its pluggable architecture. In addition to providing [80+ Lua plugins](https://apisix.apache.org/plugins/) out of the box, APISIX also supports external plugins written in other languages through [plugin runners](https://apisix.apache.org/docs/go-plugin-runner/getting-started/) and [WebAssembly (Wasm)](https://apisix.apache.org/docs/apisix/next/wasm/).
```

Comment:
> https://apisix.apache.org/docs/apisix/next/wasm/

Don't you think it would be better to use "https://apisix.apache.org/docs/apisix/wasm/"? Because the current hyperlink is for the "next" version.